### PR TITLE
Setting up DB connection

### PR DESCRIPTION
**Pull request by tharathara**
add db connection files

### CODE REVIEW

File: DBconnection.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var express = require('express');
+var mysql = require('mysql');
+
+var connection = mysql.createConnection({
+
+        host: '127.0.0.2',
```
Comment (miyurusankalpa) [inline DBconnection.js:6]: please use env file to enter environment variables

File: package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```
Comment (miyurusankalpa) [inline package.json:5]: there isnt seem to be a index.js in this repo

**Review by miyurusankalpa — CHANGES_REQUESTED** [review]
Please add the index.js as well

**Comment by tharathara** [comment]
@miyurusankalpa  noted :D thanks

**Pull request merged**